app/src/main/res/values-nl/strings.xml

```diff
@@ -0,0 +1,259 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="header_playlists">Afspeellijsten</string>
+    <string name="header_songs">Nummers</string>
+    <string name="header_artists">Artiesten</string>
+    <string name="header_albums">Albums</string>
+    <string name="header_genres">Genres</string>
+    <string name="header_now_playing">Nu aan het spelen</string>
+    <string name="header_about">Over</string>
+    <string name="action_settings">Instellingen</string>
+    <string name="header_settings">Instellingen</string>
+    <string name="header_search">Zoeken</string>
+    <string name="header_all_music">Bibliotheek</string>
+    <string name="header_browser">Navigeren</string>
+    <string name="header_recently_added">Recent Toegevoegd</string>
+    <string name="header_all_songs">Alle Nummers</string>
+    <string name="playlist">Afspeellijst</string>
+    <string name="header_create_playlist">Creëer afspeellijst</string>
+    <string name="playlist_auto">Slimme Afspeellijst</string>
+    <string name="header_add_queue_to_playlist">Voeg wachtwij toe aan afspeellijst</string>
+    <string name="playlist_auto_new">Nieuwe Slimme Afspeellijst</string>
+    <string name="playlist_auto_choose_by">Kies nummers met</string>
+    <string name="first_launch_title">Welkom bij Jockey</string>
+    <string name="action_read_privacy_policy">Open</string>
+    <string name="alert_privacy_policy_updated">Jockey\'s privacy policy is gewijzigd</string>
+    <string name="action_refresh">Bibliotheek herladen</string>
+    <string name="action_sort">Sorteer...</string>
```

Comment:
Can we replace the `...`'s with `&#8230;`? This will use the ellipses character instead, which looks slightly nicer and is consistent with the other translations. (Same thing applies in several other places throughout this file)

Reply:
Fixed.